recipes/pynamical/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pynamical" %}
+{% set version = "0.1" %}
+{% set sha256 = "79c72b15b043f40ee41d5ce4bcd35bb9c08f70ccff5f6e93ec66df18e39faea9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - matplotlib
+    - pandas
+    - numba
+
+test:
+  imports:
+    - pynamical
+
+about:
+  home: https://github.com/gboeing/pynamical
+  license: MIT
+  license_family: MIT
```

Comment:
Please add a `MANIFEST.in` to your project and please add `include LICENSE.txt` to the `MANIFEST.in`.